From ba45c057de6e8da9e3ebd54d416866c04d09496d Mon Sep 17 00:00:00 2001
From: Sertonix <sertonix@posteo.net>
Date: Tue, 8 Apr 2025 11:26:45 +0200
Subject: [PATCH] cpio: map -F to --file long option

function                                             old     new   delta
.rodata                                           100437  100444      +7
------------------------------------------------------------------------------
(add/remove: 0/0 grow/shrink: 1/0 up/down: 7/0)                 Total: 7 bytes
---
 archival/cpio.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/archival/cpio.c b/archival/cpio.c
index f0d990048..521001921 100644
--- a/archival/cpio.c
+++ b/archival/cpio.c
@@ -421,6 +421,7 @@ int cpio_main(int argc UNUSED_PARAM, char **argv)
 #endif
 #endif
 		"owner\0"        Required_argument "R"
+		"file\0"         Required_argument "F"
 		"verbose\0"      No_argument       "v"
 		"null\0"         No_argument       "0"
 		"quiet\0"        No_argument       "\xff"
-- 
2.49.0

